crates/nu_plugin_polars/src/dataframe/command/aggregation/count.rs

```diff
@@ -0,0 +1,71 @@
+use crate::dataframe::values::NuExpression;
+use crate::values::{cant_convert_err, CustomValueSupport, PolarsPluginObject, PolarsPluginType};
+use crate::PolarsPlugin;
+use nu_plugin::{EngineInterface, EvaluatedCall, PluginCommand};
+use nu_protocol::{Category, Example, LabeledError, PipelineData, ShellError, Signature, Type};
+
+pub struct ExprCount;
+
+impl PluginCommand for ExprCount {
+    type Plugin = PolarsPlugin;
+
+    fn name(&self) -> &str {
+        "polars count"
+    }
+
+    fn description(&self) -> &str {
+        "Returns the number of non-null values in the column."
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .input_output_types(vec![(
+                Type::Custom("expression".into()),
+                Type::Custom("expression".into()),
+            )])
+            .category(Category::Custom("dataframe".into()))
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        // to add an example we will need to be able to
+        // allow null values to be entered into the dataframe from nushell
+        // and retain the correct dtype. Right now null values cause the dtype to be object
+        vec![]
```

Comment:
maybe it's better to remove examples() than have vec![]? i could go either way.

Reply:
I almost removed it, I left it there as a reminder because I want to revisit the conversions to allow null/nan values. I also wanted to leave an explanation why there currently isn't an example.